src/Scoreboard.cpp

```diff
@@ -416,6 +416,22 @@ cTeam * cScoreboard::GetTeam(const AString & a_Name)
 
 
 
+AStringVector cScoreboard::GetTeamNames()
+{
+	AStringVector Teams;
+
+	for (auto it : m_Teams)
```

Comment:
Don't use `it` for things that are not iterators. I think `for (const auto & Team: m_Teams)` would be better.